discord/activity.py

```diff
@@ -674,6 +674,11 @@ def album_cover_url(self):
     def track_id(self):
         """:class:`str`: The track ID used by Spotify to identify this song."""
         return self._sync_id
+    
+    @property
+    def track_url(self):
+        """:class:`str`: The track URL to listen on Spotify."""
+        return 'https://open.spotify.com/track/' + self.track_id
```

Comment:
```suggestion
        return f'https://open.spotify.com/track/{self.track_id}'
```

Reply:
Following syntax of line671 for uniformity.